config.json

```diff
@@ -479,7 +479,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+            "algorithms",
+            "mathematics",
+            "loops",
+            "arrays"
```

Comment:
Could you please change it to `lists`. 
In my mind `arrays` can store data only of the same type (in Java for example), but `lists` can store data of any type (like in Python or any implementation of `List` interface in Java)